src/config.cpp

```diff
@@ -13,8 +13,20 @@
 // limitations under the License.
 
 #include "config.hpp"
+#include "log.hpp"
+#include "unit.hpp"
 
 #include <cassert>
+#include <cstdlib>
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <unordered_map>
+#include <vector>
+
+#ifdef _WIN32
```

Comment:
I think you did not end up using Windows specific code. If that is correct, remove it.